src/algorithms/policy_gradient/sac.jl

```diff
@@ -117,8 +112,8 @@ end
 This function is compatible with a multidimensional action space.
 """
 function evaluate(p::SACPolicy, state)
-    μ, log_σ = p.policy(state)
-    π_dist = Normal.(μ, exp.(log_σ))
+    μ, logσ = p.policy(state)
+    π_dist = StructArray{Normal}((μ, exp.(logσ)))
```

Comment:
I saw that StructArrays were used in PPO so I assume there is some benefit from them so I added them here also.

Reply:
Well, as seen in the failing CI this may create some problems because the gradient calculation includes the `evaluate` function and not all types play well with Zygote. Im not sure if this unknown performance gain is worth the effort to define an adjoint.

Reply:
Yeah, realised that too. Removed it.